lib/std/json.zig

```diff
@@ -1531,6 +1531,40 @@ fn parseInternal(comptime T: type, token: Token, tokens: *TokenStream, options:
                 else => return error.UnexpectedToken,
             }
         },
+        .Vector => |vectorInfo| {
+            switch (token) {
+                .ArrayBegin => {
+                    var r: T = undefined;
+                    var i: usize = 0;
+                    errdefer {
+                        while (true) : (i -= 1) {
+                            parseFree(vectorInfo.child, r[i], options);
+                            if (i == 0) break;
+                        }
+                    }
+                    while (i < vectorInfo.len) : (i += 1) {
+                        r[i] = try parse(vectorInfo.child, tokens, options);
+                    }
+                    const tok = (try tokens.next()) orelse return error.UnexpectedEndOfJson;
+                    switch (tok) {
+                        .ArrayEnd => {},
+                        else => return error.UnexpectedToken,
+                    }
+                    return r;
+                },
+                .String => |stringToken| {
```

Comment:
Do we really want to parse a string into a vector?

Reply:
Why not?
It will allow fast string processing in future. :)